layout-management-webapps/src/main/webapp/vue-app/site-management/initComponents.js

```diff
@@ -28,7 +28,7 @@ const components = {
   'site-management-sites-list': SitesList,
   'site-management-site-card': SiteCard,
   'site-management-site-card-menu': SiteCardMenu,
-  'site-card-properties-drawer': SiteCardPropertiesDrawer ,
+  'site-properties-drawer': SiteCardPropertiesDrawer ,
```

Comment:
```suggestion
  'site-properties-drawer': SitePropertiesDrawer ,
```

Rename SiteCardPropertiesDrawer.vue to SitePropertiesDrawer.vue